### ISSUE TITLE
Uncaught TypeError: window.wrap is not a function

### ISSUE THREAD

**Issue by noraj** [mythsman/hexo-filter-asciinema#4]
![image](https://user-images.githubusercontent.com/16578570/108186303-7d8f7800-710d-11eb-994b-3727ddd1b9fd.png)

![image](https://user-images.githubusercontent.com/16578570/108186315-81bb9580-710d-11eb-923c-3687e6d55950.png)

![image](https://user-images.githubusercontent.com/16578570/108186383-9861ec80-710d-11eb-8f46-e83bbb997cda.png)

**Comment by noraj** [comment]
Upstream issue: https://github.com/asciinema/asciinema-player/issues/145

**Comment by noraj** [comment]
Workaround:

```markdown
<script>var wrap = undefined;</script>
[@asciinema](/casts/tmpoht4gen6-ascii.cast)
```

**Comment by noraj** [comment]
@mythsman Why did you closed it? I see no fix in https://github.com/mythsman/hexo-filter-asciinema/commits/master

**Comment by mythsman** [comment]
I just thought you don't need my fix any more , since it's been too loog .

Now , let me fix it ...

**Comment by mythsman** [comment]
I have fixed it at version 1.0.6 , so I'm gonna close this issue.
